ansible/roles/schulcloud-server-core/templates/tldraw-migrate-job.yml.j2

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: tldraw-migration-job
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: tldraw-migration
+spec:
+  template:
+    metadata:
+      labels:
+        app: tldraw-migration
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+        runAsNonRoot: true
+      containers:
+      - name: tldraw-migration-job
+        image: {{ SCHULCLOUD_SERVER_IMAGE }}:{{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        imagePullPolicy: IfNotPresent
+        envFrom:
+        - configMapRef:
+            name: api-configmap
+        - secretRef:
+            name: api-secret
+        - secretRef:
+            name: api-files-secret
+        - secretRef:
+            name: tldraw-server-secret
```

Comment:
add a comment that this is just for this job and should not be an example for anyone else